src/test/resources/regressions/features/ghost_field/ghost-field-comparison-simple01.gobra

```diff
@@ -0,0 +1,39 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+// tests that Gobra correctly excludes ghost fields when comparing structs using the standard Go comparison
+
+package GhostFieldComparisonSimple01
+
+type Test struct {
+    actualField int
+    ghost ghostField int
+}
```

Comment:
Please also check fields that are structs with ghost fields.

Reply:
I've added a corresponding testcase and actually uncovered & fixed an issue